docs/app-dev-context/side-effects.md

```diff
@@ -0,0 +1,20 @@
+---
+id: side-effects
+title: Side Effects
```

Comment:
Should we combine this file with `\concepts\what-is-a-side-effect` and then use the one file both here and in the Glossary?